test/RequestableERC20.js

```diff
@@ -0,0 +1,135 @@
+// require('openzeppelin-test-helpers/configure')({ web3 });
+const { expectEvent, expectRevert } = require('openzeppelin-test-helpers');
+const { padLeft, padRight } = require('./helpers/pad');
+const chai = require('chai');
+
+const RequestableERC20 = artifacts.require('./RequestableERC20.sol');
+
+chai.use(require('chai-bn')(web3.utils.BN));
+
+const { expect } = chai;
+
+const toBN = web3.utils.toBN;
+
+contract('RequestableERC20', (accounts) => {
+  const owner = accounts[0];
+  const user = accounts[1];
+
+  const tokenAmount = web3.utils.toBN(1000000);
+  const requestAmount = web3.utils.toBN(100);
+
+  let token;
+
+  before(async () => {
+    token = await RequestableERC20.deployed();
+
+    console.log(`token:    ${token.address}`);
+
+    await token.mint(user, tokenAmount, {from: owner});
+  });
+
+  describe('Valid Mint Token by Owner', async () => {
+    it('Check Balance of User', async () => {
+      expect(await token.balanceOf(user)).to.be.bignumber.equal(tokenAmount);
+    });
+  });
+
+  describe('Invalid Mint Token by User', async () => {
+    it('Mint Token User by self', async () => {
+      await expectRevert.unspecified(
+        token.mint(user, tokenAmount, {from: user})
+      );
+    });
+  });
+
+  describe('Add an Address into minters', async () => {
+    it('append 0x00..01 into minters array)', async () => {
+       const nullNA = "0x0000000000000000000000000000000000000001"
+       await token.addMinter(nullNA);
+       expect(await token.isMinter(nullNA)).to.be.equal(true);
+    });
+  });
+
+  describe('Check Ownership as Minter', async () => {
+    it('renounce Minter(removed minter list)', async () => {
+      await token.renounceMinter();
+      await expectRevert.unspecified(
+        token.mint(user, tokenAmount, {from: owner})
+      );
+    });
+  });
+
+  describe('Start Enter and Exit test', () => {
```

Comment:
I think it will be more readable to construct test case as below

- describe: Request for [requetable-variable]
  - describe: Enter
    - it: test cases...
  - describe: Exit
    - it: test cases...